mime/multipart.ts

```diff
@@ -245,13 +245,13 @@ function skipLWSPChar(u: Uint8Array): Uint8Array {
 }
 
 export interface MultipartFormData {
-  file(key: string): FormFile | FormFile[] | undefined;
-  value(key: string): string | undefined;
+  files(key: string): FormFile[] | undefined;
+  values(key: string): string[] | undefined;
```

Comment:
This is going to be a breaking change...

How about aligning `value()` method to the current `file()` method like `value(): string | string[] | undefined`?

And in that case, `entries` would be:
```ts
function* entries(): IterableIterator<
  [string, string | string[] | FormFile | FormFile[] | undefined]
> { ... }
```

Reply:
That is what the original PR did in the first commit and over conversations in Discord I suggested going this route.

I don't think overloading the return type is a good idea, actually let me word that more strongly, it's terrible and I'm okay with making this a breaking change.

Reply:
I did exactly that in `89229de`, but I agree with caspervonb on breaking this to make it better. everyone should know that std is brittle right now anyway and imports are versioned. So overall I don't think that should be an issue.

Reply:
OK. I'm convinced. Maybe it's good time to break things, and existing users should be protected by versioning.